Applications/Spire/Source/Ui/InputWidget.cpp

```diff
@@ -0,0 +1,22 @@
+#include "Spire/Ui/InputWidget.hpp"
+#include "Spire/Spire/Dimensions.hpp"
+
+using namespace Spire;
+
+InputWidget::InputWidget(QWidget* parent)
+    : QLineEdit(parent) {
+  setStyleSheet(QString(R"(
```

Comment:
Not enough functionality here to justify a class. If you want you can write a function that applies a certain style to a class.

Reply:
So like:

// Ui.cpp

apply_line_edit_style(QWidget* widget) {
    static auto style = ...;
    widget->setStylesheet(style);
}